### PR TITLE
Database-setup

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_12_12_132914) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "authors", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```
Comment (Kwaku28) [inline db/schema.rb:20]: [OPTIONAL]

I suggest that you do away with the `authors` table and include it as a column in the `books` table. In creating a book the essential parameters will be the name and the author, so including the author in the `books` table will be easier than having a table to record authors.

**Review by Kwaku28 — CHANGES_REQUESTED** [review]
Hello @Patfarmurs,

Great work so far with your project. I believe this is a positive start and I look forward to the final stage of the project.


## Highlights
✔️ Good Effort
✔️ Positive start

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is based on personal references. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # https://guides.rubyonrails.org/configuring.html#database-pooling
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: Thyshaln08
```
[the remark this answers is not in the document]
Comment (Patfarmurs) [reply config/database.yml:24]: Hi @Kwaku28  thanks for the suggestion, I'll create the env file for the authentication

**Review by Kwaku28 — APPROVED** [review]
Hello @Patfarmurs,

Good job implementing the changes. I wish you the best in the rest of your work.

Cheers, happy coding.

**Comment by Patfarmurs** [comment]
Hello @Kwaku28 , thanks a lot for your time I appreciate that.

**Pull request merged**